sklearn/cluster/spectral.py

```diff
@@ -42,7 +80,6 @@ def discretize(vectors, copy=True, max_svd_restarts=30, n_iter_max=20,
         Determines random number generation for rotation matrix initialization.
         Use an int to make the randomness deterministic.
         See :term:`Glossary <random_state>`.
-
```

Comment:
Here is the problem. Add a white line.  This Fix the Travis Error. 

I test it, and this was the problem

Reply:
Great, thanks very much! Fixed and ready to go.